tools/publish-jar-local.sh

```diff
@@ -0,0 +1,52 @@
+set -euo pipefail
+
+if [[ -f "$HOME/.jabba/jabba.sh" ]]; then
```

Comment:
For making sure that the correct version of java is used, I personnally use a different approach: instead of forcing the use of `jabba` (which is not necessarily required for that, the user may want to use something else), I just ensure the current version in the env is the correct one and otherwise I suggest to use jabba. Example from [this script](https://github.com/LedgerHQ/ledger-wallet-daemon/blob/db58572c98ccfe8869eec46510abb3984c48f69a/tooling/clitools/wd/src/core_build_command.sh#L34):

```bash
IFS=. read major minor patch <<<$(java -version 2>&1 | awk -F '"' '/version/ {print $2}')
if [ $major -ne 1 ] | [ $minor -ne 8 ]; then
    printf "Openjdk should match version 1.8\n\n"
    printf "Tip ! you can use:\n"
    printf "$ jabba use adopt@1.8.0-292\n"
    exit 1
fi
```

Reply:
I'm not saying my approach is better, just a suggestion :)

Reply:
Thanks !